custom_components/tuya_local/devices/blitzwolf_bw_lt31.yaml

```diff
@@ -0,0 +1,155 @@
+name: Blitzwolf BW-LT31 LED Strip
+products:
+  - id: 66072501f4cfa2fe2887
+    name: Blitzwolf BW-LT31 LED Strip
+primary_entity:
+  entity: light
+  icon: "mdi:led-strip-variant"
+  dps:
+    - id: 20
+      type: boolean
+      name: switch
+    - id: 21
+      type: string
+      name: color_mode
+      mapping:
+        - dps_val: dynamic_mod
+          value: Dynamic
+        - dps_val: colour
+          value: hs
+        - dps_val: scene_mod
+          value: Scene
+        - dps_val: music
+          value: Music
+    - id: 22
+      name: brightness
+      type: integer
+      range:
+        min: 10
+        max: 1000
+      mapping:
+        - dps_val: null
+          value: 0
+        - scale: 3.92
+    - id: 24
+      name: rgbhsv
+      type: hex
+      format:
+        - name: h
+          bytes: 2
+          range:
+            min: 0
+            max: 360
+        - name: s
+          bytes: 2
+          range:
+            min: 0
+            max: 1000
+        - name: v
+          bytes: 2
+          range:
+            min: 0
+            max: 1000
+    - id: 102
+      name: music_data
+      type: integer
+    - id: 103
+      name: line_sequence_adjustment
+      # 线序调整
+      type: integer
+      hidden: true
```

Comment:
It doesn't make sense for these to be hidden, and they are not surfaced any other way.